### PR TITLE
Add moderntribe/tribe-common-style dependency

### PR DESCRIPTION
**Pull request by lucatume**
It can now be installed as a source repository using:

```bash
composer require --dev moderntribe/tribe-common-styles --prefer-source
```

### CODE REVIEW

File: src/Tribe/Views/V2/Template.php [hunk 1]
```diff
@@ -106,4 +106,53 @@ public function get_base_template_file() {
...
+	}
+
+	/**
+	 * Returns the template global values only.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative array containing only the template global values.
+	 */
+	public function get_global_values() {
```
Comment (bordoni) [inline src/Tribe/Views/V2/Template.php]: Should we define these on the common class?

File: src/Tribe/Views/V2/Template.php [hunk 2]
```diff
@@ -106,4 +106,53 @@ public function get_base_template_file() {
...
+	}
+
+	/**
+	 * Returns the template local values only.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative array containing only the template local values.
+	 */
+	public function get_local_values() {
```
Comment (bordoni) [inline src/Tribe/Views/V2/Template.php]: Should we define these on the common class?

File: src/Tribe/Views/V2/Template.php [hunk 3]
```diff
@@ -106,4 +106,53 @@ public function get_base_template_file() {
...
+	}
+
+	/**
+	 * Returns the template local values merged over the global ones.
+	 *
+	 * @since TBD
+	 *
+	 * @return array An associative array of the template global values.
+	 */
+	public function get_values() {
```
Comment (bordoni) [inline src/Tribe/Views/V2/Template.php]: Should we define these on the common class?

File: src/Tribe/Views/V2/Template.php [hunk 4]
```diff
@@ -106,4 +106,53 @@ public function get_base_template_file() {
...
+	 * Sets an array of values in the template.
+	 *
+	 * This is a sugar method to calling `Tribe\Events\Views\V2\Template::set` in a loop.
+	 *
+	 * @since TBD
+	 *
+	 * @param  array    $values    An associative array of keys and values that will be set on the template.
+	 * @param  boolean  $is_local  Set the values in the local or Global context.
+	 */
+	public function set_values( array $values, $is_local = true ) {
```
Comment (bordoni) [inline src/Tribe/Views/V2/Template.php]: Should we define these on the common class?

File: src/Tribe/Views/V2/Template.php (hunk 1 above)
Comment (lucatume) [reply src/Tribe/Views/V2/Template.php]: committed by mistake

**Pull request merged**